gnomad/assessment/sanity_checks.py

```diff
@@ -104,78 +116,83 @@ def make_filters_sanity_check_expr(
 
 
 def sample_sum_check(
-    ht: hl.Table,
-    prefix: str,
+    t: Union[hl.MatrixTable, hl.Table],
+    subset: str,
     label_groups: Dict[str, List[str]],
-    verbose: bool,
-    subpop: bool = None,
     sort_order: List[str] = SORT_ORDER,
-) -> None:
+    delimiter: str = "-",
+    metric_first_label: bool = True,
+) -> Tuple[dict, dict]:
```

Comment:
PART OF LARGE SUGGESTION

```suggestion
) -> dict:
```